test/bridge/openbabel.py

```diff
@@ -0,0 +1,11 @@
+from cclib.parser import ccData
+from cclib.bridge import cclib2openbabel
+
+
+def test_uracil():
+    data = ccData(cclib2openbabel.readfile("../data/misc/uracil.xyz", "XYZ"))
```

Comment:
I'm getting an error here; I think the path should be ```"../../data/misc/uracil.xyz"```.

Can you also rename the Python test file? It was causing some weird problems with importing openbabel due to the name.